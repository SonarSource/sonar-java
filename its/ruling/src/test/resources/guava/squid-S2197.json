{
'com.google.guava:guava:src/com/google/common/io/BaseEncoding.java':[
823,
],
}
