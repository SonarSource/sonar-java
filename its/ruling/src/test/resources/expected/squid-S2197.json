{
'project:javax/swing/plaf/basic/BasicTableHeaderUI.java':[
103,
],
}
